sdk/servicebus/tests.yml

```diff
@@ -5,6 +5,4 @@ extends:
   parameters:
     MaxParallel: 4
     ServiceDirectory: servicebus
-    TimeoutInMinutes: 120
-    EnvVars:
-      SERVICE_BUS_CONNECTION_STRING: $(net-service-bus-test-connection-string)
```

Comment:
Once these changes are confirmed, I'll clean up the test resources area where the existing static namespace exists, as well as the CI environment and variable group.

Reply:
Azure resources and build pipeline variables have been cleaned up.